scio-neo4j/src/it/scala/com/spotify/scio/neo4j/Neo4jIOIT.scala

```diff
@@ -6,22 +6,28 @@ import org.apache.beam.runners.direct.DirectRunner
 import org.apache.beam.sdk.options.PipelineOptionsFactory
 import org.neo4j.driver.{AuthTokens, Driver, GraphDatabase}
 import org.scalatest.concurrent.Eventually
-
-import java.time.Year
+import org.testcontainers.utility.DockerImageName
 
 import scala.jdk.CollectionConverters._
 
 object Neo4jIOIT {
+
+  val ImageName: DockerImageName = {
+    val tag = classOf[Driver].getPackage.getImplementationVersion.take("M.m.p".length)
+    DockerImageName.parse("neo4j").withTag(tag)
```

Comment:
version is now taken from the drive. @turb can your try if this works on M1 ?

Reply:
It seems to work on the Docker side, however I get:

```
[info] - should read cypher query from the graph database *** FAILED ***
[info]   java.lang.IllegalArgumentException: unable to serialize DoFnWithExecutionInformation{doFn=org.apache.beam.sdk.io.neo4j.Neo4jIO$ReadFn@2b3d73e2, mainOutputTag=Tag<output>, sideInputMapping={}, schemaInformation=DoFnSchemaInformation{elementConverters=[], fieldAccessDescriptor=*}}
[info]   at org.apache.beam.sdk.util.SerializableUtils.serializeToByteArray(SerializableUtils.java:59)
[info]   at org.apache.beam.repackaged.direct_java.runners.core.construction.ParDoTranslation.translateDoFn(ParDoTranslation.java:736)
<snip>
[info]   ...
[info]   Cause: java.io.NotSerializableException: com.spotify.scio.neo4j.Neo4jIOIT$$anonfun$1$$anonfun$2
[info]   at java.base/java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1175)
<snip, same for write>
```

I tried force `rowMapper` and `paramsBuilder` to be static by putting it in the object, but that did not work.

Reply:
Fixed with 49362e2c5a48cbdeabb9a7a231276c9948cc55b3

Reply:
I pushed my refactoring too quickly without testing. We need to make the `RowMapper` and `ParametersBuilder` serializable. sorry about that